requirements/requirements-base.txt

```diff
@@ -0,0 +1,9 @@
+djangorestframework
+django-rest-swagger
+django-websocket-redis
+redis
+premailer
+
+# push notifications
+django-push-notifications==1.4.1
```

Comment:
Do not freeze version on pypi packages. It is making a lot problems. This is not recommended pip.